pkgs/development/libraries/ffmpeg/generic.nix

```
@@ -108,6 +109,7 @@
   withNvdec ? withHeadlessDeps && withNvcodec,
   withNvenc ? withHeadlessDeps && withNvcodec,
   withOpenal ? withFullDeps, # OpenAL 1.1 capture support
+  withOpenapv ? withHeadlessDeps && stdenv.hostPlatform.isLinux && lib.versionAtLeast version "8.0", # OpenAPV encoding support
```

This also builds fine on darwin (tested on aarch64-darwin).

```suggestion
  withOpenapv ? withHeadlessDeps && lib.versionAtLeast version "8.0", # OpenAPV encoding support
```

---

okay, added the guard for this library since upstream only officially supports linux. will remove that, thanks for testing!